### DISCUSSION TITLE
V for particle Physicist and other Root users

### DISCUSSION THREAD

**Discussion by Yuval91** [vlang/v#14555] [Ideas]
Hi,
I am very excited about the V project and I believe it could be a very good substitute for the fast-decaying but still very powerful [Root](https://root.cern/) framework. 
This is the main platform particle Physicist in Cern and Farmi-lab and other institutes around the world use for their computations. 
The problem with Root is that its have to be very fast, therefore it is written as a c++ library, with a clang-based interpreter. 
There is also a python library but the Root-python interface is very slow and prone to errors.

In any case, if V will pick up the challenge it will get massive support from a relatively small (~10K people) but very organized and experience community of users and developers. 

Good luck!

**Comment by JalonSolov** [comment]
@ulises-jeremias can likely best address this.

**Comment by ulises-jeremias** [comment]
I think it is a good goal to have. Trying to solve that kind of problems involves a lot of different areas like Distributed Programming, Scientific Computing, among others. We are already working on increasing the ecosystem we have and it will be needed to think in providing things such us ROOT does.

We will keep you updated with new improvements and milestones we complete to get closer to that goal!
